prusa/link/web/wifi-wizard/src/App.svelte

```diff
@@ -0,0 +1,316 @@
+                            Disconnecting
+                        {/if}
+                    {:else if ap.saved}
+                        Saved
+                    {/if}
+                </span>
+            </div>
+            <div class="col-auto" style="width:125px">
+                    <span>{ap.frequency}</span>
+            </div>
+            <div class="col-auto" style="width: 150px">
+                <button class="btn btn-outline-light float-right">Details</button>
+            </div>
+            <div class="w-100"></div>
+            {#if selectedSSID === ap.ssid }
+                <div class="container row" transition:slide>
+                    <div class="col-auto" style="width: 60px;"></div>
+                    {#if ap.saved}
+                        <div class="col container">
+                            <div class="row pt-2 pd-2 input-group">
+                                {#if ap.state >= 1 && ap.state <= 3}
+                                    <form class="col-auto" action="/wifi/api/disconnect" method="post" data-action="{actions.disconnect}" on:submit|preventDefault={(e) => {connectionChange(e, ap)}}>
+                                        <input type="hidden" name="ssid" value="{ap.ssid}">
+                                        <input class="btn btn-outline-light" type="submit" value="Disconnect">
+                                    </form>
+                                {:else}
+                                    <form class="col-auto" action="/wifi/api/connect" method="post" data-action="{actions.connect}" on:submit|preventDefault={(e) => {connectionChange(e, ap)}}>
+                                        <input type="hidden" name="ssid" value="{ap.ssid}">
+                                        <input class="btn btn-outline-light" type="submit" value="Connect">
+                                    </form>
+                                {/if}
+                                <form class="col-auto" action="/wifi/api/forget" method="post" data-action="{actions.forget}" on:submit|preventDefault={(e) => {connectionChange(e, ap)}}>
+                                    <input type="hidden" name="ssid" value="{ap.ssid}">
+                                    <input class="btn btn-outline-light" type="submit" value="Forget">
+                                </form>
+                            </div>
+                        </div>
+                    {:else}
+                        <div class="col-md-5 col-lg-4 col container">
+                            <ConnectForm {ap} {connectionChange}/>
+                        </div>
+                    {/if}
+                    <div class="col-md"></div>
+                </div>
+            {/if}
+        </div>
+    {/each}
+</div>
+
+<h2 style="margin-bottom:0.2em;">Connect to another network</h2>
+<div class="container p-0" transition:slide>
+    <div class="row">
+        <div class="col-lg-4 col">
+            <ConnectForm ap={ {} } {connectionChange}/>
```

Comment:
Instead of `ap={ {} }`

You can setup default value in ConnectForm component
export let ap = {};

Or make it working with undefined value